### ISSUE TITLE
Add "Wishlist" Button for Products

### ISSUE THREAD

**Issue by mrvishalsharma01** [DharshiBalasubramaniyam/ice-cream-parlour-website#88]
We need to implement a "Wishlist" feature that allows users to save products they like for future reference. This feature will enhance the user experience by enabling users to easily find and purchase their favorite items later. can i work on this issue? can you assign me this?

**Comment by DharshiBalasubramaniyam** [comment]
@mrvishalsharma01, Can you develop wishlist like the cart model which is currently in the website?

**Comment by mrvishalsharma01** [comment]
Yes, I can try to develop a wishlist similar to the cart model on the website

**Comment by mrvishalsharma01** [comment]
please assign me this

**Comment by SumitSahu44** [comment]
assign this to me i will do it ASAP under gssoc-ext label

**Comment by DharshiBalasubramaniyam** [comment]
@mrvishalsharma01 any updates?

**Comment by Codersweta7** [comment]
@DharshiBalasubramaniyam 
You can assign this to me ...
